### PR TITLE
feat: Camp overview page - edit camper information

### PR DESCRIPTION
**Pull request by dhmi19**
* Added menu to camp overview registration table

## Notion ticket link

[Camp Overview: Edit Camper Information](https://www.notion.so/uwblueprintexecs/Camp-Overview-Edit-Camper-Information-e1dd8cd472ef4ad6a2de3cf1026e01fa)

## Implementation description
* Added a new component called `CampersTableMenu` for the dropdown menu in the registered campers table 

## Steps to test
1. go to http://localhost:3000/admin/camp/63139c7bc3d7b55b44a01531 (this is the url to access the Guelph Camp overview page)
2. click on the 3 dot menu to see all the menu options

## What should reviewers focus on?
* Design specifications 
* How do you make the menu only render downwards towards the bottom? It always renders upwards for me :( 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx [hunk 1]
```diff
@@ -211,12 +216,7 @@ const CampersTable = ({
                     padding="0px"
                     maxWidth="32px"
                   >
-                    <IconButton
-                      aria-label="Mark as active button"
-                      icon={<FaEllipsisV />}
-                      variant=""
-                      // onClick={() => console.log("3 dot button pressed!")}
-                    />
```
Comment (e-wai) [inline frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx:219]: Hmm not sure why the linter isn't catching this one, but are there unused `IconButton` and `FaEllipsisV` imports now?

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import {
+  Menu,
+  MenuButton,
+  IconButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+import React from "react";
```
Comment (e-wai) [inline frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx:8]: super super nit: make React the first import

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { FaEllipsisV } from "react-icons/fa";
+
+const CampersTableMenu = (): JSX.Element => {
+  return (
+    <Menu placement="bottom-end">
+      <MenuButton as={IconButton} icon={<FaEllipsisV />} variant="" />
+      <MenuList minW={0} width="164px" p={0}>
```
Comment (e-wai) [inline frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx:15]: why the `p={0}` attribute?

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <MenuItem height="48px">View details</MenuItem>
+        <MenuItem height="48px">Move camper</MenuItem>
+        <MenuItem height="48px">Edit</MenuItem>
+        <MenuItem
+          height="48px"
+          textColor="secondary.critical.100"
+          onClick={() => console.log("hi")}
```
Comment (e-wai) [inline frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx:22]: Are you planning to handle the remove in this component? I'm thinking there should be a passed function as a parameter into this child component from the parent, which would handle the close modal + open confirmation

Comment (e-wai) [reply frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx:22]: Same thing, will need callbacks for the other 3 menu items too?

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  Menu,
+  MenuButton,
+  IconButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+import React from "react";
+import { FaEllipsisV } from "react-icons/fa";
+
+const CampersTableMenu = (): JSX.Element => {
```
Comment (e-wai) [inline frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx:11]: nit: thoughts on renaming to `CampersTableKebabMenu`, which makes it very explicit that it's the stacked dots menu?

**Comment by e-wai** [comment]
Can you include video demo in PR description? Giving ✅ to unblock this, just had some small + quick comments

> How do you make the menu only render downwards towards the bottom? It always renders upwards for me :(

will play around on my local and see what it looks like for me, surprised the `placement` attribute wasn't enough

**Review by e-wai — APPROVED** [review]

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx (hunk 1 above)
Comment (dhmi19) [reply frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx:219]: yeah that's strange, I ran lint fix but i guess it didn't remove them

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx (hunk 3 above)
Comment (dhmi19) [reply frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx]: removed

File: frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx (hunk 4 above)
Comment (dhmi19) [reply frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx]: add props to the table and a quick draft of the edit modal to experiment with how the callbacks would trigger the modal

**Pull request merged**